### ISSUE TITLE
Conditional color

### ISSUE THREAD

**Issue by notepaddo** [RealRaven2000/QuickFolders#339]
Hello, I would like my Quickfolders to have different colors depending on having any message or having unread messages. For example, if the folder doesn’t contain any message, it could be automatically grey; if it contains messages, it could become automatically green; if it contains unread messages, it could become red. Is this possible? Or perhaps is it a feature already present in QuickFolders and I just need to learn how to make it work?

**Comment by RealRaven2000** [comment]
No this feature doesn't currently exist - the colors currently are used to highlight individual folders and also the status of current folder and drag status (when you drag an email over a tab). The coloring system is already quite complicated so this would need to be well considered.

**Comment by notepaddo** [comment]
Is it possible to have an option to see in bold not only the tabs that have unread messages, but also the tabs that just contain messages, even if they are marked as read?

**Comment by RealRaven2000** [comment]
> Is it possible to have an option to see in bold not only the tabs that have unread messages, but also the tabs that just contain messages, even if they are marked as read?

No, that's not possible - but you can display all tabs that have messages in them in italic font:
![image](https://user-images.githubusercontent.com/6314674/232849367-05b04fe7-958d-4f58-959c-464d87ad789c.png)

this does include unread folders as well. 

But if you enable that option you might be able to re-style it via userChrome.css:

```css
#QuickFolders-Toolbar toolbarbutton.has-messages {
  font-style: normal !important; 
  font-weight: bold;
}
```

**Comment by notepaddo** [comment]
Thank you very much! It works!

**Comment by pgnd** [comment]
@RealRaven2000 

i _think_ this is a close-enuf thread ...

can QF folder-tab styling also be changed based on message tag types ?

e.g., in addition to knowing that a folder-tab contains new/unread messages, with appropriate change in tab styling &/or display of new/unread message counts, i'd like to be able to know/see that there are "Important" messages (e.g., if tagged by a sieve rule based on content, sender, etc) in the folder.

Additional/different coloring (or icon? or blinking?)  on tab would be useful.

I'm finding users note that there are new/unread messages, but due to simple overload, that doesn't get attention sufficiently to get them to OPEN a folder and pay attention to a message.

If a message is tagged "from management", or somesuch, having more urgent notification/flags should help ...

**Comment by RealRaven2000** [comment]
> @RealRaven2000
> can QF folder-tab styling also be changed based on message tag types ?
> 
> e.g., in addition to knowing that a folder-tab contains new/unread messages, with appropriate change in tab styling &/or display of new/unread message counts, i'd like to be able to know/see that there are "Important" messages (e.g., if tagged by a sieve rule based on content, sender, etc) in the folder.
> 
> Additional/different coloring (or icon? or blinking?) on tab would be useful.
> 
interesting suggestion, although complete information overload for the average user. Feel free to open another feature request, but it's a little dangerous, may mean a lot of work for me and performance may potentially be bad. I am using Thunderbird's built in functions for retrieving the number of mails / unread mails, which highly likely are cashed / updated when new mail is fetched. Counting / searching / gathering tags would have to be done by my Add-on separately so it might bog down the performance especially if you use IMAP.
